examples/website-demos/social-messaging/src/components/ChannelContainer/ChannelContainer.tsx

```diff
@@ -1,18 +1,36 @@
 import React from 'react';
 
+import { MessageInput, Window } from 'stream-chat-react';
+
+import { NewChat } from '../NewChat/NewChat';
 import { SocialChannelHeader } from '../ChannelHeader/SocialChannelHeader';
 import { SocialMessageList } from '../MessageList/SocialMessageList';
 
 import './ChannelContainer.scss';
 
-type Props = {};
+type Props = {
+  isNewChat: boolean;
+  setNewChat: React.Dispatch<React.SetStateAction<boolean>>,
+};
 
 export const ChannelContainer: React.FC<Props> = (props) => {
+  const { isNewChat, setNewChat } = props;
+  
+  const ChannelContents = () => {
```

Comment:
does this need to be broken out into it's own component?

Reply:
Nope, I can change this. I originally was going to have multiple headers here and was trying to keep it clean, but instead I'm dynamically changing the title of the one header.